framework/doc/content/source/timeintegrators/AStableDirk4.md

```diff
@@ -1,33 +1,45 @@
 # AStableDirk4
 
-!alert! construction title=Undocumented Class
-The AStableDirk4 has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
+!syntax description /Executioner/TimeIntegrator/AStableDirk4
 
-```markdown
-# AStableDirk4
+This method can be expressed as a Runge-Kutta method with the following Butcher Tableau:
 
-!syntax description /Executioner/TimeIntegrator/AStableDirk4
+!equation
+\begin{array}{c|ccc}
+\gamma & \gamma & 0 & 0 \\
+1/2 & 1/2 - \gamma & \gamma & 0 \\
+1 - \gamma & 2 \gamma & 1 - 4 \gamma & \gamma \\
+\hline
+    &  \dfrac{1}{24(1/2-\gamma)} & 1 - \dfrac{1}{12(1/2-\gamma)^2} & \dfrac{1}{24(1/2-\gamma)^2}
+\end{array}
 
-## Overview
+where $\gamma = 1/2 + \dfrac{\sqrt{3}}{3} \cos( \dfrac{\pi}{18}) \approx 1.06857902130162881$
 
-!! Replace these lines with information regarding the AStableDirk4 object.
+The stability function for this method is:
 
-## Example Input File Syntax
+!equation
+R(z) = \dfrac{-0.76921266689461 z^3 - 0.719846311954034 z^2 + 2.20573706179581 z - 1.0}{
+       1.22016884572716 z^3 - 3.42558337748497 z^2 + 3.20573706551682 z - 1.0}
 
-!! Describe and include an example of how to use the AStableDirk4 object.
 
-!syntax parameters /Executioner/TimeIntegrator/AStableDirk4
+The method is *not* L-stable; it is only A-stable:
 
-!syntax inputs /Executioner/TimeIntegrator/AStableDirk4
+!equation
+\lim_{z->\infty} R(z) = -0.630414937726258
 
-!syntax children /Executioner/TimeIntegrator/AStableDirk4
-```
-!alert-end!
+## Notes:
+- Method is originally due to [!cite](crouzeix1975)
```

Comment:
Needs blank line before

Reply:
And delete the blank line after this item.

Reply:
Also it's weird to have a colon after a subsection name

Reply:
fixed